tests/settings/ConfigurationGatheringTest.py

```diff
@@ -70,15 +70,16 @@ def test_gather_configuration(self):
 
         self.assertEqual(
             str(sections['cli']),
-            "cli {bears : 'JavaTestBear', config : " + repr(temporary) +
+            "cli {bears : 'JavaTestBear', config : " +
+            PathArg(repr(temporary)) +
```

Comment:
Must be `repr(PathArg(temporary))`

This is only coincidentally working because of the remaining `escape(temporary, ...)` above